examples/04_GSD_FeI2.jl

```diff
@@ -131,6 +143,11 @@ for _ in 1:10_000
     step!(sys_large, langevin)
 end
 
+# With this increase in temperature, the suggested timestep has increased slightly.
+
+suggest_timestep(sys_large, langevin; tol=1e-2)
+langevin.Δt = 0.040;
```

Comment:
I think this interface is good. I guess the pattern is new in Sunny: allowing a field to be left unset in a struct and requiring that it be set before use. But I think I prefer this to, say, having a setter function.